spec/spec_helper.rb

```diff
@@ -62,10 +62,6 @@
   #   - http://rspec.info/blog/2014/05/notable-changes-in-rspec-3/#zero-monkey-patching-mode
   config.disable_monkey_patching!
 
-  # This setting enables warnings. It's recommended, but in some cases may
-  # be too noisy due to issues in dependencies.
-  config.warnings = true
```

Comment:
This was actually the default generated `spec_helper.rb` from RSpec via `rails generate rspec:install` :8ball: